olp-cpp-sdk-authentication/src/IntrospectAppResult.cpp

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#include "olp/authentication/IntrospectAppResult.h"
+
+namespace olp {
+namespace authentication {
+IntrospectAppResult::IntrospectAppResult() noexcept {}
```

Comment:
In my opinion you can inline these as they are rather small.

Reply:
Do you mean only constructor or set/get methods too?
I think, inline them all should be fine.

Reply:
All methods. Constr/destructor declaration I would remove, else you need to apply the rule of 5 so that we can efficiently move.